src/main/app/controller/courts/AddressController.ts

```diff
@@ -170,6 +180,16 @@ export class AddressController {
     return errors;
   }
 
+  private validateDescriptionLength(address: DisplayAddress, isPrimaryAddress: boolean): string[] {
+    const errors: string[] = [];
+    if (!isPrimaryAddress) {
+      if (!(address.description?.trim().length<50) || !(address.description_cy.trim().length<50)) {
```

Comment:
There are some welsh address lines that may be longer? 

Also on the migration on the API side: 
1. we do not have a length constraint for the new field 
2. we are missing the migration for the existing courts description field to the new table (I think...double check on this one)

Reply:
I have increased the limit to 100.